src/index.d.ts

```diff
@@ -204,3 +220,49 @@ type Institution = {
   updated_at: number;
   resource_type: string;
 }
+
+type Position = {
+  _id: string;
+  class: string;
+  classes: {
+    class: string;
+    weight: number;
+  };
+  category: string;
+  categories: Array<string>;
+  security: Security;
+  investments: Array<PositionInvestment>
+  quantity: number;
+  book_value: number;
+  market_value: number;
+  gain_amount: number;
+  gain_percent: number;
+}
+
+type Security = {
+  _id: string;
+  currency: string;
+  symbol: string;
+  type: string;
+  name: string;
+  aliases: Array<string>;
+  last_price: number;
+  last_date: string;
+  high_price: number;
+  high_date: string;
+  low_price: number;
+  low_date: string;
+  cusip: string;
+}
+
+type PositionInvestment = {
+  institution: Institution;
```

Comment:
`institution: Institution`, whole Institution object or just the mongodb id?

Reply:
You are right just string